File: src/main/java/com/addressbook/model/IgniteMetricsContainer.java
package com.addressbook.model;

import com.addressbook.ignite.GridDAO;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;
import org.apache.ignite.cache.CacheMetrics;

import java.io.Serializable;
import java.util.HashMap;
import java.util.Map;

@ToString
@Getter
@Setter
public class IgniteMetricsContainer implements Serializable {

    private Map<String, IgniteCacheMetrics> igniteCacheMetricsMap;

    public IgniteMetricsContainer(){
        igniteCacheMetricsMap = new HashMap<>();
        for(Map.Entry<String, CacheMetrics> metricsEntry:  GridDAO.getCacheMetrics().entrySet()){
            igniteCacheMetricsMap.put(metricsEntry.getKey(), new IgniteCacheMetrics(metricsEntry.getValue()));
        }
    }
}

File: addressbook-main/src/main/kotlin/com/addressbook/model/IgniteMetricsContainer.kt
package com.addressbook.model

import com.addressbook.ignite.GridDAO
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.context.ApplicationContext
import org.springframework.stereotype.Component
import java.io.Serializable
import java.util.*


@Component
class IgniteMetricsContainer : Serializable {

    @Autowired
    var igniteDao: GridDAO? = null

    var igniteCacheMetricsMap: HashMap<String, IgniteCacheMetrics>? = null

    fun refresh(): IgniteMetricsContainer {
        igniteCacheMetricsMap = HashMap()
        for (metricsEntry in igniteDao?.getCacheMetrics()?.entries!!) {
            igniteCacheMetricsMap?.put(metricsEntry.key, IgniteCacheMetrics(metricsEntry.value))
        }
        return this
    }
}
